docs/intents/ref/ViewMessages.md

```diff
@@ -0,0 +1,124 @@
+---
+id: ViewMessages
+sidebar_label: ViewMessages
+title: ViewMessages
+hide_title: true
+---
+# `ViewMessages`
+
+Search and display a list of messages to the user.
+
+## Intent Name
+
+`ViewMessages`
+
+## Display Name
+
+`View Messages`
+
+## Possible Contexts
+
+* [SearchCriteria](../../context/ref/SearchCriteria)
+
+## Example
+
+Request to display messages for a specific ticker:
+```js
+const searchCriteria = {
+    type: "fdc3.searchCriteria",
+    contexts: [
+        {
+            type: "fdc3.instrument",
+            id: {
+                ticker: "AAPL"
+            }
+        }
+    ]
+}
+
+fdc3.raiseIntent('ViewMessages', searchCriteria);
+```
+
+Request to display messages for a specific **contact**:
+
+```js
+const searchCriteria = {
+    type: "fdc3.searchCriteria",
+    contexts: [
+        {
+            type: "fdc3.contact",
+            name: "Jane Doe",
+            id: {
+                email: "jane.doe@mail.com"
+            }
+        }
+    ]
+}
+
+fdc3.raiseIntent('ViewMessages', searchCriteria);
+```
+
+Request to display messages for a specific **organization**:
```

Comment:
Probably redundant. It'd be more succinct to add a line of copy either at the start of the example section or under possible contexts explaining that any appropriate contexts (or combination of contexts!) can be added to the search criteria.